### PR TITLE
list all possible strings for schema.pre

### PR DESCRIPTION
**Pull request by IslandRhythms**

### CODE REVIEW

File: docs/middleware.md [hunk 1]
```diff
@@ -118,6 +118,17 @@ schema.pre('save', function(next) {
 });
 ```
 
+Here are the possible strings that can be passed to `pre()`
```
Comment (vkarpov15) [inline docs/middleware.md:121]: You're missing a whole bunch. `estimatedDocumentCount`, `findOneAndReplace`, `aggregate`, `insertMany`, and more.

Also, please put this list immediately after "* [insertMany](api/model.html#model_Model-insertMany)" on L69

**Review by vkarpov15 — CHANGES_REQUESTED** [review]

**Comment by jdevries3133** [comment]
Hey folks, is it possible to extend an opportunity to @larasodjati to draft a PR for the issue she's identified here?

Lara is a student in an org I'm volunteering for and I'm hoping to sponsor / assist her to see this issue through to a merged PR.

Obviously totally understand if you want to close this out quickly, and @IslandRhythms, your work is definitely appreciated and I wouldn't want to step on your toes either. In any case, it would be great if we could open an opportunity for Lara to make her first open source contribution!

Thanks all!

**Comment by IslandRhythms** [comment]
I have no objection but I'll need to confirm with val if its ok.

**Comment by IslandRhythms** [comment]
Val would rather me do it, sorry.

**Review by vkarpov15 — APPROVED** [review]
Thanks :+1:

**Comment by vkarpov15** [comment]
@jdevries3133 sorry we missed your post, this PR is already done. But I appreciate the offer. Please let me know if you want another light docs issue that your student can help out with.

**Review by vkarpov15 — APPROVED** [review]
Thanks :+1:

**Comment by jdevries3133** [comment]
> @jdevries3133 sorry we missed your post, this PR is already done. But I appreciate the offer. Please let me know if you want another light docs issue that your student can help out with.

No worries, totally understand. We'll definitely be on the lookout for another light docs issue, if you have a suggestion that would be much appreciated!

**Pull request merged**